### DISCUSSION TITLE
Новые архитектуры для docker

### DISCUSSION THREAD

**Discussion by aspeed** [mikopbx/Core#778] [Ideas (Идеи)]
Добрый день!
Планируюся ли образы docker, отличные от x86_64? Очень интересует aarch64 (arm64)!

**Comment by eggplants** [comment]
> Added Docker container version for ARM64

https://github.com/mikopbx/Core/releases/tag/2024.2.28-dev

**Reply by eggplants** [reply]
Currently, it is not published on ghcr.io.
